src/translations/en/publicationTypes.json

```diff
@@ -10,7 +10,7 @@
   "DegreeMaster": "Master thesis",
   "DegreePhd": "Doctoral thesis",
   "OtherStudentWork": "Other student thesis",
-  "Journal": "Contribution to journal",
+  "Journal": "Contribution to journal/magazine/newspaper",
```

Comment:
Benytte seg av Oxford comma istedet for "/"? Bedre for skjermlesere.
ex.: "journal, magazine, or newspaper"

Reply:
Dette kommer rett fra PO, men enig med deg, så kan gjøre en dobbelsjekk der

Reply:
Enighet om å bruke komma!